### ISSUE TITLE
About the training data.

### ISSUE THREAD

**Issue by mikebartorn** [jrgillick/laughter-detection#17]
I have a question about the training data, because you mentioned in section 3.4 of the paper that the training Audioset dataset does not mark the start and end times of laughter, but the Audioset test dataset does mark the start and end times , so how did you calculate the accuracy when testing?

**Comment by jrgillick** [comment]
Hi Mike, we manually annotated start and end times for 1000 files in AudioSet as part of our research, which we used for testing. We made those available here: [annotations](https://github.com/jrgillick/laughter-detection/tree/master/data/audioset/annotations/ )

To train using AudioSet, we had to train with imperfect labels: we just assumed that the entire 10 seconds of a clip contained laughter, even though we know that's not really the case.
